Eastern-Research-Group/mywaterway
# Feature/149 update pad us service

## Related Issues:
* https://jira.epa.gov/browse/HMW-149

## Main Changes:
* Switched to the latest version of the PAD-US service. 

## Steps To Test:
1. Navigate to http://localhost:3000/community/170402060107/protect
2. Expand the "Protected Areas" accordion
3. Expand all of the items under "Protected Areas"
4. Verify that all of the attributes are displayed. The attribute name used for "Public Access" was changed, so pay special attention to this field.
5. Click on these features on the map and repeat step 4 on the popups.

## TODO:
